src/index.ts

```diff
@@ -163,18 +166,22 @@ class Analytics {
   processIdentifyUserQueue() {
     this.identifyQueue.process(numberOfConcurrentJob, async (job, done) => {
       const userPayload = job.data;
-      await this.postUser(userPayload);
-
-      done();
+      try {
+        await this.postUser(userPayload);
+      } finally {
+        done();
+      }
     });
   }
 
   processTrackDataQueue() {
     this.trackQueue.process(numberOfConcurrentJob, async (job, done) => {
       const userPayload = job.data;
-      await this.postUser(userPayload);
-
-      done();
```

Comment:
in this way if get error and don't call `done` the queue would stuck so we should call `done` even if the job has been failed

Reply:
Ok ok, got it!